db/migrate/20201012120000_update_comment_to_vms_table_format.rb

```diff
@@ -0,0 +1,5 @@
+class UpdateCommentToVmsTableFormat < ActiveRecord::Migration[5.2]
+  def up
+    change_column_comment :vms, :format, "The format of the VM's disk, such as tier1."
```

Comment:
@jaywcarman can you change the comment to something like `"The format of the VM's disk, such as vmdk, qcow2, and tier1.`

I don't think parsing a column comment is going to work better than other options so the list doesn't have to be exhaustive but it should be descriptive.

Reply:
Really we just need to indicate that this column is no longer "deprecated" (even though it never really was)

Reply:
I've updated the comment to include the other example values, along with deprecating the deprecation :laughing:

Reply:
:rofl: perfect!